_posts/2021-09-02-gsoc-final-report-estefania.md

```diff
@@ -0,0 +1,299 @@
+cbar.ax.tick_params(labelsize=4, width=0.5)
+cbar.set_label('H (nm$^{-1}$)', fontsize=6, labelpad=2) 
+```
+
+<img src = "https://ojeda-e.github.io/assets/images/mycontours_dark.png" 
+style="float: center" alt = "contours" width="1000"/>
+
+Mean curvature plots provide information about the "inverted shape" of the surface, 
+which in this example is derived from phospholipids headgroups in each leaflet.
+Positive mean curvature indicates valleys (red coloured), negative mean curvature is associated
+with peaks (blue coloured). Regions where _H=0_ indicates a flat region (white coloured). In the
+example considered here, the contour plot of mean curvature shows:
+
+* Coupling between leaflets. Regions of positive curvature (red coloured) in the
+  lower leaflet match those in the upper leaflet. The same is observed for
+  regions of negative curvature (blue coloured).
+
+* A central region of positive curvature. For both upper and lower leaflet,
+  there is a central region of positive mean curvature (red coloured) along the
+  _y_ axis, while regions of negative curvature (blue coloured) are localized in
+  the bulk of the membrane, in particular bottom left and upper right regions.
+
+
+# In progress
+Currently, we are working on implementing interpolation as an option for the
+user [#52].
+
+In some situations, when selecting a very high number of bins in the grid, or
+when having regions of the grid with low sampling we may find regions of undefined values.
+For example, think of a membrane-protein system, where the bins occupied by the protein won't be
+populated by lipids, and therefore, will have a region of undefined values in
+the grid. Such undefined values spread in the array during the calculation of
+curvature, which may result in meaningless output.
+
+By adding an optional interpolation, we will be able to patch up undefined values 
+in bins inside the embedded element (i.e. protein). With this improvement, calculation
+of membrane curvature won't be hamstrung by the presence of undefined values in the grid.
+
+# What's next?
+There is always room for improvement, and MembraneCurvature is not an exception.
+One of the main limitations of the current version of MembraneCurvature is the
+inability to calculate curvature in systems like vesicles, capsids, or micelles.
+This would definitely be a nice improvement for a future release of
+MembraneCurvature!
+
+![vesicles](https://ojeda-e.github.io/assets/images/vesicles.png)
+
+We acknowledge that scientific research would benefit from a
+tool to calculate membrane curvature in these types of systems, so we are
+considering possible approaches to include more topologies in MembraneCurvature!
+
+# Acknowledgments
+Participating in GSoC with MDAnalysis has been a unique experience. I had the
+opportunity to learn best practices in software development mentored by a group
+of incredibly talented people:
+- [@lilywang](https://github.com/lilyminium) (Lily),
```

Comment:
just use the GH handle and it will automatically resolve, i.e.
```markdown
@lilyminium (Lily), @IAlibay (Irfan), ...
```
and note that Lily's handle is `@lilyminium` !